docs/topics/auth/passwords.txt

```diff
@@ -98,7 +98,7 @@ To use Argon2 as your default storage algorithm, do the following:
             'django.contrib.auth.hashers.Argon2PasswordHasher',
             'django.contrib.auth.hashers.PBKDF2PasswordHasher',
             'django.contrib.auth.hashers.PBKDF2SHA1PasswordHasher',
-            'django.contrib.auth.hashers.BCryptSHA256PasswordHasher',
+            'django.contrib.auth.hashers.BCryptSHA256PasswordHasher'
```

Comment:
Unrelated changes, should be reverted.

Reply:
I will push my basic edits in 10 minutes.